src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -301,6 +305,46 @@ static ConfigDef newConfigDef() {
             6,
             ConfigDef.Width.NONE,
             SNOWFLAKE_ROLE)
+        .define(
```

Comment:
can you add a validator for this? you can take a look at allowed values enum for `INGESTION_METHOD`

feel free to do it in separate PR since this PR is blowing up. just add a TODO with jira link.

Reply:
Added [jira](https://snowflakecomputing.atlassian.net/browse/SNOW-189106)